packages/@uppy/utils/src/Translator.ts

```diff
@@ -7,7 +7,8 @@ export interface Locale<T extends number = number> {
 
 type Options = {
   smart_count?: number
-  [key: string]: string | number | undefined
```

Comment:
I changed this on purpose as the test was failing.

Reply:
We don't want `undefined` to be a valid value.